pyproject.toml

```diff
@@ -16,7 +16,9 @@ version = "0.4.0"
 [tool.poetry.dependencies]
 python = "^3.9"
 vcrpy = "^6.0.2"
+urllib3 = "^1.26.19"
```

Comment:
can we have this as >= vs ^
this would allow urllib 2 in some python environments

Reply:
yes, of course. how silly of me.